Create Contra Case Study (#5143)
PostHog/posthog.com: contents/customers/contra.md

---
title: How Contra used session recordings to improve registrations by 30%
customer: Contra
logo: ../images/customers/contra/contra_logo.png
featuredImage: ../images/customers/contra/contra_featured.png
industries:
    - Saas
users:
    - Product
    - Engineering
toolsUsed:
    - Feature Flags
    - Session Recording
    - Product Analytics
---

> Contra is a freelance marketplace service which empowers its community of independent creatives and engineers to find work that fits the way they want to live. Since 2022 [Contra](http://www.contra.com) has used PostHog Cloud to power many of its engineering and product decisions, though previously the stack was a lot more complex.

Reviewer: ```suggestion
Contra is a freelance marketplace for service independent creatives and engineers. It empowers its community to find work that fits the way they want to live. Since 2022 [Contra](http://www.contra.com), has used PostHog Cloud to power many of its engineering and product decisions, though previously the stack was a lot more complex.
```

> Contra is a freelance marketplace for service independent creatives and engineers. It empowers its community to find work that fits the way they want to live. Since 2022 [Contra](http://www.contra.com), has used PostHog Cloud to power many of its engineering and product decisions, though previously the stack was a lot more complex.

Reviewer 2: Remove word "service" - we don't use this term

Author: ```suggestion
Contra is a freelance marketplace for independent creatives and engineers. It empowers its community to find work that fits the way they want to live. Since 2022 [Contra](http://www.contra.com), has used PostHog Cloud to power many of its engineering and product decisions, though previously the stack was a lot more complex.
```

> “Originally we were using multiple platforms,” said Allison Nulty, Head of Product at Contra. “That meant Mixpanel for analytics, LaunchDarkly for feature flags, Segment for front-end tracking and FullStory for recordings. In isolation they could work well, but they were costly even then.”

Reviewer: ```suggestion
“Originally we were using multiple platforms,” says Allison Nulty, Head of Product at Contra. “That meant Mixpanel for analytics, LaunchDarkly for feature flags, Segment for front-end tracking and FullStory for recordings. In isolation they could work well, but they were costly even then.”
```

Take or leave, feels a bit less formal using 'says'

> As a result of spreading functionality across multiple tools, Contra also faced data inconsistencies. Tools such as Segment were helpful for forwarding data between platforms, but also added to the costs and upkeep required. Eventually, Contra decided to consolidate the stack as much as possible — and that’s where PostHog came in. 

Reviewer 2: One note is that we currently still use Segment and are happy with the service - I might add that Segment was useful for tracking _and_ forwrding

Author: ```suggestion
As a result of spreading functionality across multiple tools, Contra also faced data inconsistencies. Tools such as Segment were helpful tracking for forwarding data between platforms, but also added to the costs and upkeep required. Eventually, Contra decided to consolidate the stack as much as possible — and that’s where PostHog came in. 
```

### Using feature flags for a ‘slow roll-out’ strategy

Switching to PostHog enabled Contra to consolidate feature flags, session recordings, product analytics and more into a single tool — but it also enabled the company to come together around a single platform as well. Instead of each team in the company relying on a different software, everyone was able to look at a single source of truth.

<BorderWrapper>
    <Quote
        imageSource="/images/customers/allison.jpg"
        size="md"
        name="Allison Nulty"
        title="Head of Product, Contra "

>         quote={`“A real positive has been the ability to talk to the PostHog engineers, to share feedback with them and have it received well. I think how rewarding that open line of communication is when I engage with our users too.”`}

Reviewer 2: Updated (to sound better): “A huge competitive advantage has been the ability to talk directly with the PostHog technical team via Slack. We can actively share feedback, ask questions, and make feature requests  and always see a quick turnaround time and thoughtful suggestions. At Contra, we apply these same principles of collaborating with our community to build our product and so we appreciate that level of service when on the other side.

Author: ```suggestion
        quote={`“A huge competitive advantage has been the ability to talk directly with PostHog engineers over Slack. We share feedback, ask questions, and make requests and always see a quick response time and thoughtful suggestions. At Contra, we apply these same principles of collaborating with our community to build our product.”`}
```

    />
</BorderWrapper>

> “Previously the teams all used different tools,” said Allison. “That led to a lot of confusion, because we were onboarding teams to all these different tools and it was clearly creating complexity and leading to inconsistencies…We had issues with LaunchDarkly especially, and feature flags are very important to us.”

Reviewer 3: ```suggestion
“Previously the teams all used different tools,” said Allison. “That led to a lot of confusion, because we were onboarding teams to all these different tools and it was clearly creating complexity and leading to inconsistencies… We had issues with LaunchDarkly especially, and feature flags are very important to us.”
```

> “Previously the teams all used different tools,” said Allison. “That led to a lot of confusion, because we were onboarding teams to all these different tools and it was clearly creating complexity and leading to inconsistencies… We had issues with LaunchDarkly especially, and feature flags are very important to us.”

Reviewer: ```suggestion
“Previously the teams all used different tools,” said Allison. “That led to a lot of confusion, because we were onboarding teams to all these different tools and it was clearly creating complexity and leading to inconsistencies... We had issues with LaunchDarkly especially, and feature flags are very important to us.”
```

> “Previously the teams all used different tools,” said Allison. “That led to a lot of confusion, because we were onboarding teams to all these different tools and it was clearly creating complexity and leading to inconsistencies... We had issues with LaunchDarkly especially, and feature flags are very important to us.”

Reviewer 2: I'd prefer to not callout one product that we had issues with if its not backed in data. The more specific issue we had is that "our feature flag platform (Launch Darkly) was not directly integrated with our analytics platform (Mixpanel) or session recordings (Fullstory) which made it difficult to use feature flag data to better understand user journeys and dropoff points.

Author: ```suggestion
“Previously the teams all used different tools,” said Allison. “That led to a lot of confusion, because our feature flag platform didn't integrate with our analytics platform, or our session recording tool. That made it difficult to use feature flag data to build better user journeys or understand drop-offs.”
```

> Feature flags in particular are important because the engineering team deploys all new product features behind flags and follows a careful ‘slow roll-out’ strategy. All features are first deployed to beta users, then 20% of all users, followed by 50% of all users and finally 100% of all users. Switching feature flags to PostHog enables Contra to continue testing with this method, while also analyzing feature adoption and performance in the same tool. 

Reviewer 2: rollout

### Using session recordings to improve a product

> The combination of feature flags and session recordings has been especially powerful for Contra, as it enables product managers to directly observe how user behavior changes between variants. In other instances, session recording alone enables the team to identify where to make improvements. 

Reviewer 2: product managers and designers. 

What are we getting at with this ` In other instances, session recording alone enables the team to identify where to make improvements. ` ? 

I think its moreso that we leverage session recording to both observe the impact of new features as well as debug user experiences. For example, if we have a notification that a user failed to setup payments, we can go to their Posthog recordings and identify the issue ourselves, allowing us to proactively engage with the user. This means faster debugging and a better user experience. 

Feel free to include ^

Reviewer 2: I think the value here is that our support team also leverages this tool

Author: ```suggestion
The combination of feature flags and session recordings has been especially powerful for Contra, as it enables product managers and designers to directly observe how user behavior changes between variants. In other instances, session recording alone enables the team to identify where to make improvements.

"For example," said Allison, "if we have a notification that a user failed to setup payments, we can view their recordings in PostHog and identify the issue ourselves, enabling us to proactively engage with the user. This means faster debugging and a better user experience. Our support team can also leverage recordings in a similar way."
```

![Contra screenshot](../images/customers/contra/contra_session.gif)
<Caption>Using PostHog session recording to analyze user sign-ups</Caption>

> “We had a waitlist for one feature,” said Allison. “We weren’t seeing expected results, so we used PostHog to look at the funnel and where people were dropping off. We could jump from there straight to the session recordings and that’s when we saw there was a button missing. That meant we were missing a huge number of sign-ups.”

Reviewer 2: We were able to easily go from a conversion funel in the analytics tool to session recordings for that dropoff point to identify why users were not converting to the waitlist. With this analysis, we identified one path where users were not seeing a reminder modal and therefore missing out on a large number of waitlist entries.

Author: ```suggestion
“We used PostHog to look at our funnel for a waitlist in the insights tool,” said Allison. “From there, we could easily jump to session recordings to see the drop-off point. With this analysis, we identified one path where users were not seeing a reminder modal and therefore we were missing out on a large number of waitlist entries.”
```

After noticing the issue, the team made a change and saw an immediate, massive improvement in conversion. Today, 30% of waitlist sign-ups are attributed to this new and improved flow. 

> “Adoption has even come at the encouragement of our CEO and co-founder,” said Allison. “We now have cross-functional meetings where we engineers, product managers and designers look at successful and unsuccessful sessions together to identify patterns — and they’re all familiar with the same tool.”

Reviewer 2: “Adoption has also come at the encouragement of our CEO and co-founder,” 

We have cross-functional meetings where we engineers, product managers, and designers "watch the tapes" to observe successful and unsuccessful sessions and identify patterns. Everyone at Contra is familiar with PostHog.

Author: ```suggestion
“Adoption has also come at the encouragement of our CEO and co-founder,” said Allison. “We have cross-functional meetings where engineers, product managers, and designers 'watch the tapes' to observe successful and unsuccessful sessions and identify patterns. Everyone at Contra is familiar with PostHog.”
```

> “That ability to consolidate platforms, to not have to onboard teams to six different platforms, to be able to easily connect events and funnels to recordings? It was a real a-ha moment for me!”

Reviewer 2: “That ability to consolidate platforms, to simplify team onboarding, to be able to easily connect events and funnels to recordings? It was a real a-ha moment for me!”

Author: ```suggestion
“That ability to consolidate platforms, to simplify team onboarding, to be able to easily connect events and funnels to recordings? It was a real a-ha moment for me!”
```

---
title: How Contra used session recordings to improve registrations by 30%
customer: Contra
logo: ../images/customers/contra/contra_logo.png
featuredImage: ../images/customers/contra/contra_featured.png
industries:
    - Saas
users:
    - Product
    - Engineering
toolsUsed:
    - Feature Flags
    - Session Recording
    - Product Analytics
---

Contra is a freelance marketplace for independent creatives and engineers. It empowers its community to find work that fits the way they want to live. Since 2022 [Contra](http://www.contra.com), has used PostHog Cloud to power many of its engineering and product decisions, though previously the stack was a lot more complex.

> “Originally we were using multiple platforms,” said Allison Nulty, Head of Product at Contra. “That meant Mixpanel for analytics, LaunchDarkly for feature flags, Segment for front-end tracking and FullStory for recordings. In isolation they could work well, but they were costly even then.”

Reviewer 2: In isolation they could worked well, but they were costly and complex.

Author: ```suggestion
“Originally we were using multiple platforms,” said Allison Nulty, Head of Product at Contra. “That meant Mixpanel for analytics, LaunchDarkly for feature flags, Segment for front-end tracking and FullStory for recordings. In isolation they could work well, but they were costly and complex.”
```

> As a result of spreading functionality across multiple tools, Contra also faced data inconsistencies. Tools such as Segment were helpful tracking for forwarding data between platforms, but also added to the costs and upkeep required. Eventually, Contra decided to consolidate the stack as much as possible — and that’s where PostHog came in. 

Reviewer 2: Tools such as Segment were helpful tracking for forwarding data between platforms, but also added to the costs and upkeep required.

Not sure I follow this comment ^ we still use Segment today

Author: ```suggestion
As a result of spreading functionality across multiple tools, Contra also faced data inconsistencies. Each tool had it's own costs and required its own upkeep. Eventually, Contra decided to consolidate the stack as much as possible — and that’s where PostHog came in. 
```

    />
</BorderWrapper>

“Previously the teams all used different tools,” said Allison. “That led to a lot of confusion, because our feature flag platform didn't integrate with our analytics platform, or our session recording tool. That made it difficult to use feature flag data to build better user journeys or understand drop-offs.”

Feature flags in particular are important because the engineering team deploys all new product features behind flags and follows a careful ‘slow rollout’ strategy. All features are first deployed to beta users, then 20% of all users, followed by 50% of all users and finally 100% of all users. Switching feature flags to PostHog enables Contra to continue testing with this method, while also analyzing feature adoption and performance in the same tool. 

### Using session recordings to improve a product

The combination of feature flags and session recordings has been especially powerful for Contra, as it enables product managers and designers to directly observe how user behavior changes between variants. In other instances, session recording alone enables the team to identify where to make improvements.

> "For example," said Allison, "if we have a notification that a user failed to setup payments, we can view their recordings in PostHog and identify the issue ourselves, enabling us to proactively engage with the user. This means faster debugging and a better user experience. Our support team can also leverage recordings in a similar way."

Reviewer 2: ur support team can also leverage recordings in a similar way." --> "This has become foundational to our customer support processes."

Author: ```suggestion
"For example," said Allison, "if we have a notification that a user failed to setup payments, we can view their recordings in PostHog and identify the issue ourselves, enabling us to proactively engage with the user. This means faster debugging and a better user experience. This has become foundational to our customer support processes too."
```

![Contra screenshot](../images/customers/contra/contra_session.gif)

> <Caption>Using PostHog session recording to analyze user sign-ups</Caption>

Reviewer 2: Really this is to analyze waitlist joins ^ not sign-ups

Author: ```suggestion
<Caption>Using PostHog session recording to analyze waitlist joins</Caption>
```
